tests/sentry/api/endpoints/test_organization_derive_code_mappings.py

```diff
@@ -62,15 +61,14 @@ def test_get_single_match(self, mock_get_trees_for_org):
     @patch("sentry.integrations.github.GitHubIntegration.get_trees_for_org")
     def test_get_start_with_backslash(self, mock_get_trees_for_org):
         file = "stack/root/file.py"
-        frame_info = FrameFilename(f"/{file}")
         config_data = {"stacktraceFilename": f"/{file}"}
         expected_matches = [
             {
                 "filename": file,
                 "repo_name": "getsentry/codemap",
                 "repo_branch": "master",
-                "stacktrace_root": f"{frame_info.root}",
-                "source_path": _get_code_mapping_source_path(file, frame_info),
+                "stacktrace_root": "",
+                "source_path": "",
```

Comment:
Our expected answers should be hard coded based on our understanding of how the logic should work - it should not use the actual logic its testing.